src/bin/server/subscription.rs

```diff
@@ -191,6 +200,9 @@ impl Subscription {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use futures::{Stream};
```

Comment:
`Future` isn't in scope here, which is required in order to use `map` on like 226 since it's a method of the `Future` trait.

Reply:
Could've sworn it was passing before. Fixed in https://github.com/fat-whale/tectonicdb/pull/14/commits/9f064fd27293e07e19ba2e6a8da59a335c14bff3